psi4/src/psi4/libfock/v.h

```diff
@@ -194,5 +194,50 @@ class UV : public VBase {
 
     void print_header() const override;
 };
+
+// => Separate Classes <= //
+class Num1Int {
```

Comment:
I don't have a great feel for C++ style preferences, but I think this class could be a separate file h/cc.